### PR TITLE
created frontend Readme.md

### PR DESCRIPTION
**Pull request by pabaross**
Created Frontend Readme.md

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+This project was bootstrapped with
+[Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
```
Comment (yuukiross) [inline frontend/README.md:7]: This is not neccessary

File: frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+You don’t have to ever use `eject`. The curated feature set is suitable for
+small and middle deployments, and you shouldn’t feel obligated to use this
+feature. However we understand that this tool wouldn’t be useful if you couldn’t
+customize it when you are ready for it.
+
+## Learn More
+
+You can learn more in the
+[Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+
```
Comment (yuukiross) [inline frontend/README.md:58]: This is not enough for description.

**Review by yuukiross — CHANGES_REQUESTED** [review]

File: frontend/README.md (hunk 1 above)
Comment (pabaross) [reply frontend/README.md:7]: ok

File: frontend/README.md (hunk 2 above)
Comment (pabaross) [reply frontend/README.md:58]: ok

**Pull request merged**